linera-summary/src/github.rs

```diff
@@ -165,17 +165,18 @@ impl Github {
     }
 
     pub async fn comment_on_pr(&self, body: String) -> Result<()> {
-        if self.is_local {
-            info!("Printing summary to stdout:");
-            println!("{}", body);
-        } else {
+        // Always print the summary to stdout, as we'll use it to set the job summary in CI.
+        info!("Printing summary to stdout...");
+        println!("{}", body);
+
+        if !self.is_local {
             info!("Commenting on PR {}", self.context.pr_number);
             self.octocrab
                 .issues(
                     self.context.repository.owner.clone(),
                     self.context.repository.name.clone(),
                 )
-                .create_comment(self.context.pr_number, body)
+                .create_comment(self.context.pr_number, body.clone())
```

Comment:
Is this `clone` needed?

Reply:
Good catch, it isn't. I think it slipped through during conflict resolution